de.dlr.sc.virsat.project.test/src/de/dlr/sc/virsat/project/test/AProjectTestCase.java

```diff
@@ -266,4 +268,26 @@ protected void assertRetry(int retries, int milliSeconds, Runnable assertRunnabl
 			} 
 		}
 	}
+	
+	
+	public static class TestUserContext implements IUserContext {
+
+		public TestUserContext(String userName, boolean su) {
+			this.userName = userName;
+			this.su = su;
+		}
+		
+		String userName;

```

Comment:
Maybe put the variables before the constructor?

Reply:
OK